<?xml version="1.0"?>
<testsuite name="Unknown Run" time="0" tests="0" failures="0" errors="0"><testsuite name="Module A" time="0" tests="2" failures="0" errors="0"><testcase name="test10" time="0"/><testcase name="test11" time="0"/></testsuite><testsuite name="Module B" time="0" tests="1" failures="0" errors="0"><testcase name="test20" time="0"/></testsuite><testsuite name="Module E" time="0" tests="1" failures="0" errors="1"><testcase name="test50" time="0"><error message="An exception was thrown!">An exception was thrown!</error></testcase></testsuite><testsuite name="Module F" time="0" tests="1" failures="1" errors="0"><testcase name="test60" time="0"><failure message="expected: 5, actual: 'six'">expected: 5, actual: 'six'</failure></testcase></testsuite></testsuite>